recipes/causalml/meta.yaml

```diff
@@ -0,0 +1,112 @@
+{% set name = "causalml" %}
+{% set version = "0.12.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/causalml-{{ version }}.tar.gz
+  sha256: 1a5216ee58522cce725907b005bf93232eb311929b36a20fbf8ee341b5781ebb
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - pytest
+    - python >=3.6
+    - cython ==0.29.23
+    - setuptools >=41.0.0
+    - numpy >=1.18.5
+    - scikit-learn >=0.22.0
+    - python-graphviz
+    - graphviz
+  run:
+    - python >=3.6
+    - setuptools >=41.0.0
+    - pip >=10.0
+    - numpy >=1.18.5
+    - scipy >=1.4.1
+    - matplotlib-base
+    - pandas >=0.24.1
+    - scikit-learn >=0.22.0
+    - statsmodels >=0.9.0
+    - seaborn
+    - cython ==0.29.23
+    - xgboost
+    - pydotplus
+    - tqdm
+    - shap <0.38.1
+    - dill
+    - lightgbm
+    - pygam
+    - packaging
+    - pytorch
+    - pyro-ppl
+    - python-graphviz
+    - graphviz
+
+test:
+  imports:
+    - causalml
+    - causalml.inference.meta
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://causalml.readthedocs.io/en/latest/?badge=latest
+  summary: Causal ML is a Python package that provides a suite of uplift modeling and causal inference methods using machine learning algorithms based on recent research. It provides a standard interface that allows user to estimate the **Conditional Average Treatment Effect** (CATE) or **Individual Treatment Effect** (ITE) from experimental or observational data.
+  license: Apache-2.0
+  license_file: LICENSE
+  description: |
+    <a href="https://github.com/uber/causalml">
```

Comment:
This is going to be hard to maintain. Do you need all this? Maybe just a shorter description.